src/dart/ast.rs

```diff
@@ -288,21 +414,36 @@ impl BoolBinOp {
 }
 
 enum_from_primitive! {
-    #[derive(Copy, Clone, PartialEq, Eq, Debug)]
-    pub enum ValueBinOp {
-        IfNull,
-        Add,
-        Sub,
-        Mul,
-        Div,
-        Mod,
-        TruncDiv,
-        Lsh,
-        Rsh,
-        BitAnd,
-        BitXor,
-        BitOr,
-    }
+/// Value binary operators.
+#[derive(Copy, Clone, PartialEq, Eq, Debug)]
+pub enum ValueBinOp {
+    /// Conditional expression (`a ?? b`).
+    /// If a is non-null, returns its value;
+    /// otherwise, evaluates and returns the value of b.
```

Comment:
Missing backticks around mentions of `a` and `b`.

Reply:
Isn't the whole expression the example? Why would `a` and `b` be separately backticked?